### PR TITLE
Add some new posibrain names

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

**Comment by optimumtact** [comment]
dumb stripper action

**Comment by Zytolg** [comment]
Is that a c*der reference?

**Comment by MrDoomBringer** [comment]
No player references are allowed under current policy, please excise all of these and use appropriate names.

Thank you

**Review by Mothblocks — APPROVED** [review]

**Review by Mothblocks — APPROVED** [review]
i approve

**Review by Fikou — CHANGES_REQUESTED** [review]
Code looks bad

**Comment by LemonInTheDark** [comment]
pepin-the-short-being-crowned-king-of-the-franks-by-st-boniface-in-751

**Comment by NamelessFairy** [comment]
https://github.com/tgstation/tgstation/pull/60608

This is a content expansion, please wait until next month 🐈

File: strings/names/posibrain.txt [hunk 1]
```diff
@@ -44,4 +44,6 @@ DUNC
 FXMC
 DRSD
 COI
-CYBR
\ No newline at end of file
+CYBR
+MOTH
+BLOK
```
Comment (tralezab) [inline strings/names/posibrain.txt:49]: this would be awesome, i think

```suggestion
BLOK
ARM
```

**Comment by Mothblocks** [comment]
i got bad news about that entire file you guys

**Comment by MrDoomBringer** [comment]
For anyone who wants to do further reading on this subject:
https://github.com/tgstation/tgstation/pull/12722
https://github.com/tgstation/tgstation/pull/19758
https://github.com/tgstation/tgstation/pull/12759

**Comment by Farquaar** [comment]
*No description provided.*

**Comment by MrDoomBringer** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

Your changelog is also inaccurate, so please update it.

File: strings/names/posibrain.txt [hunk 2]
```diff
@@ -44,4 +44,6 @@ DUNC
 FXMC
 DRSD
 COI
-CYBR
\ No newline at end of file
+CYBR
+MOTH
```
Comment (Fikou) [inline strings/names/posibrain.txt:48]:
```suggestion
MOTH
CHESSE20
```

File: strings/names/posibrain.txt [hunk 3]
```diff
@@ -44,4 +44,6 @@ DUNC
 FXMC
 DRSD
 COI
-CYBR
\ No newline at end of file
+CYBR
```
Comment (Fikou) [inline strings/names/posibrain.txt:47]:
```suggestion
CYBR
FIK
```

File: strings/names/posibrain.txt (hunk 1 above)
Comment (Fikou) [inline strings/names/posibrain.txt:49]:
```suggestion
BLOK
WLTR
```

Comment (Fikou) [reply strings/names/posibrain.txt:49]:
```suggestion
BLOX
```

Comment (Fikou) [reply strings/names/posibrain.txt:49]:
```suggestion
BLOK
LEMN
```

File: strings/names/posibrain.txt (hunk 2 above)
Comment (Mothblocks) [reply strings/names/posibrain.txt:48]: image

**Comment by optimumtact** [comment]
the next son of a windows admin who posts on my github pr is getting a ban

**Comment by Tennessee116** [comment]
holy shit I didn't realize the posibrain names were coder names thats clever

**Comment by Bawhoppen** [comment]
self-merge, dear god

**Pull request merged**